.gitignore

```diff
@@ -15,3 +15,4 @@ node_modules
 testgrounds
 generated
 yarn-error.log
+build
```

Comment:
my brain always wants an `.npmignore` with each .gitignore for an node module.   It would size down our package yes?

Reply:
Just saw this, @GantMan -- I explicitly specify what gets included in the NPM module in `package.json#files`, so an .npmignore would have no effect.